security.md

```diff
@@ -80,6 +81,21 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2020-14334"></a>CVE-2020-14334: World readable cache could expose sensitive settigs
+Even encrypted settings have their raw values cached. Too permissive mode on cache dir caused, that anyone with access to the hosting system could read this encrypted settings.
+
+*Mitigation:* override `/run/foreman` directory mode to `750`.
+To do so in a manner that survives reboot, update a file `/usr/lib/tmpfiles.d/foreman.conf`.
+```
+d /run/foreman 0750 foreman foreman -
+```
+
```

Comment:
This is not enough, the permission will be fixed on next reboot or after 24 hours when the systemd timer fires off. I suggest to also recommend changing the permissions immediately on the directory itself using `chmod`. Look:

```
[root@box lzap]# ll -ad /tmp/test
drwx------. 2 root root 40 10. srp 09.17 /tmp/test
[root@box lzap]# cat /usr/lib/tmpfiles.d/test.conf
d /tmp/test 0700 root root -
[root@box lzap]# vim /usr/lib/tmpfiles.d/test.conf
[root@box lzap]# ll -ad /tmp/test
drwx------. 2 root root 40 10. srp 09.17 /tmp/test
[root@box lzap]# systemd-tmpfiles --create
[root@box lzap]# ll -ad /tmp/test
drwxrwxrwx. 2 root root 40 10. srp 09.17 /tmp/test
```